src/components/BoemlyTabs/BoemlyTabs.tsx

```diff
@@ -1,16 +1,21 @@
 import React, { FC, useMemo, useState } from 'react';
 import { Box, Flex } from '@chakra-ui/react';
-import TabButton from './TabButton';
+import LineTabButton from './LineTabButton';
+import SolidTabButton from './SolidTabButton';
 
 export interface BoemlyTabsProps {
   tabs: { key: string; title: string; content: JSX.Element | JSX.Element }[];
   initialTabKey?: string;
+  isSolid?: boolean;
+  icon?: JSX.Element;
   onChange?: (activeTabKey: string) => void;
 }
```

Comment:
```suggestion
export interface BoemlyTabsProps {
  tabs: { key: string; title: string; content: JSX.Element | JSX.Element[], icon?: JSX.Element | JSX.Element[]; }[];
  initialTabKey?: string;
  variant?: 'line' | 'solid';
  onChange?: (activeTabKey: string) => void;
}
```

Some points:

* I think you need to add an icon to every tab. Otherwise, every tab has the same icon. 
* I would use a boolean for the `variant`. If you need a third variant in the future you will have a problem. Therefore I would use a `variant` prop with options and make one option the default one.
* `JSX.Element | JSX.Element` makes no sense → `JSX.Element | JSX.Element[]` - was an existing problem - but would be nice if it's fixed